backend/plugin/db/driver.go

```diff
@@ -158,11 +158,14 @@ type MigrationInfo struct {
 
 // ConnectionConfig is the configuration for connections.
 type ConnectionConfig struct {
-	Host     string
-	Port     string
-	Username string
-	Password string
-	Database string
+	Host string
+	Port string
+	// More hosts and ports are required by elasticsearch.
```

Comment:
It's better to use either MultiHosts or (Host, Port) instead of MultiHosts as additions. More declarative and easy to understand. This means Host and Port can be empty string.